### DISCUSSION TITLE
2023-10-16

### DISCUSSION THREAD

**Discussion by aak2075** [woowacourse-teams/2023-haru-study#656] [daily_meeting]
### 데일리 미팅

개인 일정
- 3~4시 근로: 룩소, 노아, 마코

### 오늘의 할 일

- 폴링 관련 QA

### 내일의 할 일

- 수요일까지 데모데이 소개 페이지 팀별 페이지

**Comment by yeopto** [comment]
### 체크인

10점 - 활기찬 월요일입니다.

### 체크아웃

10점 - 간만에 집중하니 좋네요

**Comment by jaehee329** [comment]
### 체크인
5점 - 미션 시간 너무 잡아먹어요 ㅠ

### 체크아웃
4점 - 미션때문에 감소

**Comment by woo-jk** [comment]
### 체크인
10점 - 주말동안 잠을 많이자서 컨디션 회복해썽요

### 체크아웃
10점 - 미션도 냈고 버그 사항도 고치고 좋아요

**Comment by woosung1223** [comment]
### 체크인
9점 - 좋아하는 일본 밴드 내한 콘서트를 가게 되어 좋습니다

### 체크아웃
9점 - 그냥 나쁘지 않네요

허브 - Long.MAX_VALUE - 테오 리뷰 받고 기분이 좋네요

**Comment by aak2075** [comment]
### 체크인

10점 - 면접 스터디 할 분 모집합니다(오늘 당장)

**Reply by MoonJeWoong** [reply]
뭐로 시작하죠?

**Reply by woo-jk** [reply]
저요

**Reply by aak2075** [reply]
cs, 자소서, 인성 등 본인이 원하는대로!

**Comment by nlom0218** [comment]
### 체크인

7점 - next가 너무 절 괴롭히네요.. 😭

### 체크아웃

9점 - 이제 조금 알듯말듯!

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 수면은 만성 부족이네요... 하지만 할 건 해야죠

### 체크아웃
9점 - 7일간 객체지향의 전사가 되겠습니다.
